man/systemd.network.xml

```diff
@@ -1141,6 +1141,39 @@ NetLabel=system_u:object_r:localnet_peer_t:s0</programlisting>
           and the reverse operation when the IPv4 address is deconfigured.</para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>IPv4NFTSet=</varname><replaceable>family</replaceable>:<replaceable>table</replaceable>:<replaceable>set</replaceable></term>
+        <term><varname>IPv6NFTSet=</varname><replaceable>family</replaceable>:<replaceable>table</replaceable>:<replaceable>set</replaceable></term>
```

Comment:
BTW, does it really make sense to have two distinct settings for IPv4 and IPv6 here? why is this in the lvalue instead of the rvalue? Can you explain? 

Can't the two kind of addresses be added to the same set? (no nft pro here, as you might guess).

it appears icky to me to allow ipv4 and ipv configuration here to be entirely orthogonal.